crates/e2e-test-utils/src/lib.rs

```diff
@@ -55,6 +56,10 @@ where
     N::AddOns: RethRpcAddOns<Adapter<N>> + EngineValidatorAddOn<Adapter<N>>,
     LocalPayloadAttributesBuilder<N::ChainSpec>:
         PayloadAttributesBuilder<<<N as NodeTypes>::Payload as PayloadTypes>::PayloadAttributes>,
+    <<N as Node<TmpNodeAdapter<N>>>::AddOns as NodeAddOns<Adapter<N>>>::Handle: RpcHandleProvider<
+        Adapter<N>,
+        <<N as Node<TmpNodeAdapter<N>>>::AddOns as RethRpcAddOns<Adapter<N>>>::EthApi,
+    >,
```

Comment:
should we make a type alias for this? Like:
```rust
type NodeAddOnHandle<N> = <<N as Node<TmpNodeAdapter<N>>>::AddOns as NodeAddOns<Adapter<N>>>::Handle;
```
and similarly for `<<N as Node<TmpNodeAdapter<N>>>::AddOns as RethRpcAddOns<Adapter<N>>>::EthApi`?

Reply:
Agreed, will clean up the trait bounds.